Installer.php

```diff
@@ -326,12 +301,15 @@ public static function setPermission(array $paths)
     }
 
     /**
-     * Generates a cookie validation key for every app config listed in "config" in extra section.
+     * Generates a cookie validation key for every application configuration in "extra" section.
      * You can provide one or multiple parameters as the configuration files which need to have validation key inserted.
      */
     public static function generateCookieValidationKey()
     {
         $configs = func_get_args();
+        if (count($configs) === 1 && is_array($configs[0])) {
+            $configs = $configs[0];
+        }
```

Comment:
What is it for?

Reply:
@samdark to support configuration in format same to `setPermission` - uniform configuration form for all methods